.github/workflows/security-code-scanner.yml

```diff
@@ -0,0 +1,43 @@
+name: 'MetaMask Security Code Scanner'
+
+on:
+  push:
+    branches: ['master']
+  pull_request:
+    branches: ['master']
+
+jobs:
+  run-security-scan:
+    runs-on: ubuntu-latest
+    permissions:
+      actions: read
+      contents: read
+      security-events: write
+    steps:
+      - name: MetaMask Security Code Scanner
+        uses: MetaMask/Security-Code-Scanner@main
+        with:
+          repo: ${{ github.repository }}
+          paths_ignored: |
+            .storybook/
+            '**/__snapshots__/'
+            '**/*.snap'
+            '**/*.stories.js'
+            '**/*.stories.tsx'
+            '**/*.test.browser.ts*'
+            '**/*.test.js*'
+            '**/*.test.ts*'
+            '**/fixtures/'
+            '**/jest.config.js'
+            '**/jest.environment.js'
+            '**/mocks/'
+            '**/test*/'
+            docs/
+            e2e/
+            merged-packages/
+            node_modules
+            storybook/
+            test*/
+          rules_excluded: example
+          mixpanel_project_token: ${{ secrets.SECURITY_CODE_SCANNER_MIXPANEL_TOKEN }}
```

Comment:
We use GA4 on the MetaMask website instead of Mixpanel. Should this be updated with the GA4 token?

Reply:
@cuonglt-ync @kevin-chassagne could one of you please review as well?

Reply:
> We use GA4 on the MetaMask website instead of Mixpanel. Should this be updated with the GA4 token?

Hi @nkkurt , it seems the MetaMask/Security-Code-Scanner Action relies on Mixpanel to log/track its own actions. So, the use of Mixpanel is not related to Metamask website, there is nothing to be worried about.

Reply:
Got it - thanks @tannn-younet !